themes/bootstrap3/js/embedded_record.js

```diff
@@ -138,6 +155,13 @@ VuFind.register('embedded', function embedded() {
               // Hide loading
               loadingNode.addClass('hidden');
               longNode.collapse('show');
+              // Setup to load a targetted tab
+              if (openTab) {
+                var tab = result.find('[id^="' + openTab + '"]');
```

Comment:
Is this selector too open-ended? It seems like there could be some edge cases where this could open the wrong tab, if you had two tabs with similar names (i.e. `hold` and `holdings`). Maybe not so likely in practice, but if something more specific could work just as well, it might be safer.